Game/level.gd

```diff
@@ -115,3 +110,23 @@ func _on_multiplayer_timer_timeout():
 	if player != null:
 		var sprite = player.get_node("AnimatedSprite2D") as AnimatedSprite2D
 		$MultiplayerController.send_player_info(Preferences.get_player_name(), sprite.animation, sprite.frame, player.position, sprite.flip_h)
+
+func _on_quit_window_close_requested():
+	get_tree().paused = false
+	$QuitMenu/QuitWindow.hide()
+
+func _on_volumeslider_value_changed(value: float):
+	$AudioStreamPlayer.set_volume_db(value - 100)
+
+func _on_color_check_toggled(toggled: bool):
+	if toggled:
+		ProjectSettings.set_setting("rendering/environment/defaults/color_blind_mode", true)
+	else:
+		ProjectSettings.set_setting("rendering/environment/defaults/color_blind_mode", false)
+
+
+func _on_dark_check_toggled(toggled: bool):
+	if toggled:
+		RenderingServer.set_default_clear_color(Color(0, 0, 0, 1))
+	else:
+		RenderingServer.set_default_clear_color(Color(1, 1, 1, 1))
```

Comment:
You need to apply the shaders to the objects in the level.  All this does currently is set the background color.

Reply:
also what do you mean by this

Reply:
@day-dreamxr maybe we could always add shaders to objects and just toggle the effect with a global uniform?
https://docs.godotengine.org/en/stable/tutorials/shaders/shader_reference/shading_language.html#global-uniforms
Either that or we'll just have to loop through every object in every group (Red/Green/Blue) and add/remove the shaders from them. I don't really know which solution is better

Reply:
> @day-dreamxr maybe we could always add shaders to objects and just toggle the effect with a global uniform? https://docs.godotengine.org/en/stable/tutorials/shaders/shader_reference/shading_language.html#global-uniforms Either that or we'll just have to loop through every object in every group (Red/Green/Blue) and add/remove the shaders from them. I don't really know which solution is better

trying to do the ladder rn and its just not going well LOLOL

Reply:
*latter